From 9b7334b61cf3277e5eb48b716f6719b4636e2572 Mon Sep 17 00:00:00 2001
From: Rich Megginson <rmeggins@redhat.com>
Date: Mon, 12 Jul 2010 17:21:01 -0600
Subject: [PATCH] Do not define SEC_SkipTemplate

Building NSS with PEM support gives an error in pbobject due to multiple
definitions of SEC_SkipTemplate.  This is already defined in libnssutil
---
 pobject.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/pobject.c b/pobject.c
index 81b9028..48f5e78 100644
--- a/mozilla/security/nss/lib/ckfw/pem/pobject.c
+++ b/mozilla/security/nss/lib/ckfw/pem/pobject.c
@@ -172,6 +172,8 @@ static const NSSItem pem_trusted = {
     (void *) &ckt_netscape_trusted, (PRUint32) sizeof(CK_TRUST)
 };
 
+/* SEC_SkipTemplate is already defined and exported by libnssutil */
+#ifdef SEC_SKIP_TEMPLATE
 /*
  * Template for skipping a subitem.
  *
@@ -182,6 +184,7 @@ static const NSSItem pem_trusted = {
 const SEC_ASN1Template SEC_SkipTemplate[] = {
     {SEC_ASN1_SKIP}
 };
+#endif
 
 /*
  * Find the subjectName in a DER encoded certificate
-- 
1.5.5.6

